cli/main.rs

```diff
@@ -0,0 +1,67 @@
+mod runtime;
+
+use clap::{command, Parser, Subcommand};
+use deno_runtime::colors;
+use deno_runtime::deno_core;
+use deno_runtime::fmt_errors::format_js_error;
+
+use deno_core::error::JsError;
+
+use runtime::AnyError;
+
+use crate::runtime::run_js_module;
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+pub struct CliArgs {
+  #[command(subcommand)]
+  command: Commands,
+}
+
+#[derive(Subcommand)]
+pub enum Commands {
+  Run {
+    /// JavaScript file containing the Station Module to run
```

Comment:
Three /// here, should be two?

Reply:
Great question!

This is a documentation comment, see https://doc.rust-lang.org/reference/comments.html#doc-comments